src/main/scala/io/sdkman/vendor/release/routes/ReleaseRoutes.scala

```diff
@@ -58,7 +58,7 @@ trait ReleaseRoutes
                     candidateO.fold(badRequestResponseF(s"Invalid candidate: ${req.candidate}")) {
                       _ =>
                         versionO.fold(
-                          saveVersion(Version(req.candidate, req.version, platform, req.url))
+                          saveVersion(Version(req.candidate, req.version, platform, req.url, req.vendor))
                             .map(_ => createdResponse(req.candidate, req.version, platform))
```

Comment:
Should we perhaps also include the vendor in the response if it is there?

Reply:
vendor is already in the version for java candidates. for example. version `15.0.1-sapmchn`